blog/2023-02-07-introducing-PR-drafting-in-reviews.md

```diff
@@ -0,0 +1,74 @@
+---
+author: Franck Nijhof
+authorURL: https://twitter.com/frenck
+authorImageURL: /img/profile/frenck.png
+authorTwitter: frenck
+title: Introducing drafting of PRs in our review process
+---
+
+:::info
```

Comment:
Feels weird ot start with an info box. Can we take this text and inject it as a new paragraph before the paragraph that starts with "We have adjusted our bot"